libtransmission/transmission.h

```diff
@@ -966,6 +966,10 @@ void tr_metainfoFree(tr_info* inf);
  */
 tr_torrent* tr_torrentNew(tr_ctor const* ctor, int* setme_error, int* setme_duplicate_id);
 
+
+/** @brief Check if uri may be interpreted as raw hash */
+bool tr_maybeHash(char const* uri);
```

Comment:
Better to leave this as private API until / unless we have non-libtransmission code using it

Reply:
> Better to leave this as private API

Current transmission GUI clients use their own version of this code for same functionality.
This code may be removed from GUI clients and replaced by library call If this commit will be accepted.